lib/graphql/tracing/data_dog_tracing.rb

```diff
@@ -15,28 +15,32 @@ class DataDogTracing < PlatformTracing
       }
 
       def platform_trace(platform_key, key, data)
-        service = options.fetch(:service, 'ruby-graphql')
+        tracer.trace(platform_key, service: service_name) do |span|
+          span.span_type = Datadog::Ext::HTTP::TYPE
 
-        pin = Datadog::Pin.get_from(self)
-        unless pin
-          pin = Datadog::Pin.new(service)
-          pin.onto(self)
-        end
-
-        pin.tracer.trace(platform_key, service: pin.service) do |span|
           if key == 'execute_multiplex'
-            span.resource = data[:multiplex].queries.map(&:selected_operation_name).join(', ')
+            operations = data[:multiplex].queries.map(&:selected_operation_name).join(', ')
+            span.resource = operations if operations.length > 0
```

Comment:
Are you sure you want to do this `.length` operation after the `join`? I realise it doesn't really matter either way, but just be aware that you're counting characters here :smile: 

Also, consider using the (perhaps more idiomatic?) `.empty?` method instead of `.length > 0`.

Reply:
I like `empty?` better!